drivers/mfd/mfd_axp192.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT x_powers_axp192
+
+#include <errno.h>
+
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(mfd_axp192, CONFIG_MFD_LOG_LEVEL);
+
+/* Chip ID value */
+#define AXP192_CHIP_ID (0x03u)
+
+/* Registers definitions */
+#define AXP192_REG_CHIP_ID (0x03u)
```

Comment:
```suggestion
#define AXP192_CHIP_ID 0x03U

/* Registers definitions */
#define AXP192_REG_CHIP_ID 0x03U
```

Reply:
fixed in [cca2db3](https://github.com/zephyrproject-rtos/zephyr/pull/58809/commits/cca2db3fdabdd7bf6802457683623f4a3ab0660d)

Reply:
please fixup original commits

Reply:
Fixed. I am still confused on force-pushing the initial commit during review.